crates/tabby-common/assets/languages.toml

```diff
@@ -41,6 +41,31 @@ top_level_keywords = [
     "return",
 ]
 
+[[config]]
+languages = ["kotlin"]
+line_comment = "//"
+top_level_keywords = [
+    "abstract",
+    "public",
+    "private",
+    "companion",
+    "class",
+    "data",
+    "const",
+    "actual",
+    "expect",
+    "sealed",
+    "suspend",
+    "fun",
+    "import",
+    "interface",
+    "package",
+    "var",
+    "val",
+    "enum",
+    "return",
```

Comment:
is return a top level keywords in kotlin?

Reply:
@wsxiaoys , I removed "return" , thanks for catching that.
Would it make sense to add:
"data class" instead of just "data"
"sealed class" instead of just "sealed"

Reply:
data / sealed should be good in this case